### ISSUE TITLE
Create button component

### ISSUE THREAD

**Issue by ostadal-vojta** [ostadal-vojta/plant-keeper#5]
There is currently button component that is not a button yet. It is just a div with hover effect.
For better accessibility this should become a button with following properties:
- `loading`
- `onClick`
- `type`
  - `primary` - full primary color
  - `secondary` - (default) - full secondary color with primary border?
  - `link` - text based button with primary color, but clickable area should be still the same as with normal buttons
  - `dashed` - transparent with secondary color dashed border
  - `ghost` - transparent with primary color border
- `danger` - this is modifier of type, replace any color with red color
- `children`

Linking [ant.design button documentation](https://ant.design/docs/spec/buttons) as inspiration/reference
